### PR TITLE
Ria 3434 pay offline by card pa

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3434](https://tools.hmcts.net/jira/browse/RIA-3434)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    void writeDecisionHearingOptionToCaseData(AsylumCase asylumCase) {
+
+        String hearingFeeOption = asylumCase
+            .read(DECISION_HEARING_FEE_OPTION, String.class).orElse("");
+
+        if (hearingFeeOption.equals(DECISION_WITH_HEARING.value())) {
+
+            asylumCase.write(
+                HEARING_DECISION_SELECTED,
+                "Decision with a hearing. The fee for this type of appeal is £140"
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java:20]: This fee of £140 can be obtained from payments-api

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            );
+
+        } else if (hearingFeeOption.equals(DECISION_WITHOUT_HEARING.value())) {
+
+            asylumCase.write(
+                HEARING_DECISION_SELECTED,
+                "Decision without a hearing. The fee for this type of appeal is £80"
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java:27]: This fee of £80 can be obtained from payments-api as this is for payment journey.
The fees are only hardcoded once in the ccd-def and its for the start appeal journey.

Comment (Lee365) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java:27]: For pay offline, payments api is not called and so the fee is hardcoded here.

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java (hunk 1 above)
Comment (Lee365) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java:20]: For pay offline, payments api is not called and so the fee is hardcoded here.

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**